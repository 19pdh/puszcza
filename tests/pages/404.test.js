import { shallowMount } from '@vue/test-utils'
import page404 from '../../pages/404'

describe('404', () => {
  // Now mount the component and you have the wrapper
  const wrapper = shallowMount(page404)

  it('match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
