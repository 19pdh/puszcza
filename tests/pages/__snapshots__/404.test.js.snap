// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 match snapshot 1`] = `
<div>
  <h1>
    404
  </h1>
   
  <p>
    Nie znaleziono strony
  </p>
</div>
`;
